src/extension/geofence/src/main/java/org/geoserver/geofence/GeofenceAccessManager.java

```diff
@@ -1084,4 +1088,43 @@ public List<Object> parseLayers(
     public int getPriority() {
         return ExtensionPriority.LOWEST;
     }
+
+    // a container for accessInfo that can hold spatial filters as Geometry type.
+    class GeoserverAccessInfo {
```

Comment:
This addition I guess was unwanted. Was removed in another pr. Please remove it.